features/exploredash/src/main/java/org/dash/wallet/features/exploredash/ui/PurchaseGiftCardFragment.kt

```diff
@@ -55,7 +58,7 @@ class PurchaseGiftCardFragment : Fragment(R.layout.fragment_purchase_gift_card)
     private val exploreViewModel: ExploreViewModel by navGraphViewModels(R.id.explore_dash) { defaultViewModelProviderFactory }
     private val enterAmountViewModel by activityViewModels<EnterAmountViewModel>()
 
-    var selectedMerchent: Merchant? = null
+    var selectedMerchant: Merchant? = null
```

Comment:
There is a `merchant` data item in the Fragment, but there is also one in the ViewModel that is assigned when the user presses "Next"

Shouldn't the merchant data be in ViewModel only?